util/paywall.go

```diff
@@ -112,6 +112,16 @@ func makeRequest(url string, timeout time.Duration) ([]byte, error) {
 	}
 	defer response.Body.Close()
 
+	if response.StatusCode != http.StatusOK {
+		body, err := ioutil.ReadAll(response.Body)
+		if err != nil {
+			return nil, fmt.Errorf("dcrdata error %v: %v %v %v", err,
```

Comment:
There is no point in printing err since it is always nil.

Reply:
This code will run only if the err is not nil. 
Did you mean body?